code/modules/modular_computers/file_system/programs/app_presets.dm

```diff
@@ -0,0 +1,99 @@
+/datum/modular_computer_app_presets
+    var/name = "default_preset"
+    var/display_name = "default preset"
+    var/description = "Description of the preset"
+    var/available = 0
+/datum/modular_computer_app_presets/proc/return_install_programs()
+    return list()
+
+/datum/modular_computer_app_presets/all
+    name = "all"
+    display_name = "All Programs"
+    description = "Contains all Progams"
+    available = 0
+/datum/modular_computer_app_presets/all/return_install_programs()
+    var/list/_prg_list = list()
+    for(var/F in typesof(/datum/computer_file/program))
+        var/datum/computer_file/program/prog = new F
+        _prg_list += prog
+    return _prg_list
+
+/datum/modular_computer_app_presets/engineering
+    name = "engineering"
+    display_name = "Engineering"
+    description = "Contains the most common engineering Programs"
+    available = 1
+/datum/modular_computer_app_presets/engineering/return_install_programs()
+    var/list/_prg_list = list(
+        new/datum/computer_file/program/power_monitor(),
+        new/datum/computer_file/program/alarm_monitor(),
+        new/datum/computer_file/program/atmos_control(),
+        new/datum/computer_file/program/rcon_console(),
+        new/datum/computer_file/program/camera_monitor/engineering(),
```

Comment:
I might change the camera monitors to only check for ID when using the application instead of checking the network access by id.
This would also entertain your edge case of TSC.
But this is out of the scope of this pull request